proto/settlus/oracle/v1alpha1/oracle.proto

```diff
@@ -23,15 +23,10 @@ message Params {
 }
 
 enum OralceTopic {
-  BLOCK = 0;
+  BLOCK = 0; // deprecated
   OWNERSHIP = 1;
 }
```

Comment:
@devbv Do you think we will ever add another oracle topic? If you think we are not going to add any new oracle topics, I will delete the `OracleTopic` enum and `NewSettlusVoteProcessors` in `x/oracle/voteprocessor/factory.go`.